test/data/manifests/rhel_87-x86_64-ec2_sap-boot.json

```diff
@@ -3825,9 +3823,6 @@
           "sha256:09149617095dc52e19cdce1e45c8245e1e92d371bd4d107320ff56788b9977f1": {
             "url": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-baseos-n8.7-20221015/Packages/libusbx-1.0.23-4.el8.x86_64.rpm"
           },
-          "sha256:0942e19ecae75b9b0f30f10c1469b783e0887728bdbd946b6e49d7c3fabc4593": {
-            "url": "https://rpmrepo.osbuild.org/v2/mirror/rhvpn/el8/el8-x86_64-appstream-n8.7-20221015/Packages/redhat-cloud-client-configuration-1-10.el8.noarch.rpm"
-          },
```

Comment:
This package must be installed. See https://github.com/osbuild/osbuild-composer/pull/3081